version.txt

```diff
@@ -1 +1 @@
-9.1.0
\ No newline at end of file
+99.1.0
```

Comment:
typo?

Reply:
nope, its just a dummy future version since this model change needs a special version from the get-go